static/app/components/searchSyntax/parser.tsx

```diff
@@ -1016,14 +1064,7 @@ export function parseSearch(
   const configCopy = {...defaultConfig};
 
   // Merge additionalConfig with defaultConfig
-  const config = mergeWith(configCopy, additionalConfig, (srcValue, destValue) => {
-    if (destValue instanceof Set) {
-      return new Set([...destValue, ...srcValue]);
-    }
-
-    // Use default merge behavior
-    return undefined;
-  });
+  const config = merge(configCopy, additionalConfig);
```

Comment:
Does this just do the same thing and I was a dum dum and missed that I could use `merge`

Reply:
No, with this change it now overwrites the sets since I need to set them to be empty sets. 
There is only one other feature setting those sets. There I manually import the defaults and merge them in.
see https://github.com/getsentry/sentry/pull/60788/files#diff-511032742590217290c98121799a6281497ad21e1d86e8abaffab1a2cb97009f

Reply:
I see, otherwise there's no way to empty out the sets right.